### PR TITLE
(docs) Add hiera-config to config docs

### PR DESCRIPTION
**Pull request by MikaelSmith**

### CODE REVIEW

File: pre-docs/bolt_configuration_options.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ ssh:
 
 `color`: Whether to use colored output when printing messages to the console.
 
+`hiera-config`: Set the path to your hiera config. The default is `hiera.yaml` inside the `Boltdir`.
```
Comment (melamos) [inline pre-docs/bolt_configuration_options.md]:
```suggestion
`hiera-config`: Specify the path to your Hiera config. The default path is `Boltdir/hiera.yaml`.
```

Comment (adreyer) [reply pre-docs/bolt_configuration_options.md]: that is not the default path. the default path is  something more like <Boltdir>/hiera.yaml

Comment (MikaelSmith) [reply pre-docs/bolt_configuration_options.md]: The phrasing `hiera.yaml inside the Boltdir` is how we describe it for other options.

**Comment by MikaelSmith** [comment]
Pushed an update with most of the changes.

**Pull request merged**